src/plus/webviews/graph/graphWebview.ts

```diff
@@ -2101,6 +2106,46 @@ export class GraphWebview extends WebviewBase<State> {
 		});
 	}
 
+	@debug()
+	private copyDeepLinkToBranch(item?: GraphItemContext) {
+		if (isGraphItemRefContext(item, 'branch')) {
+			const { ref } = item.webviewItemValue;
+			return executeCommand(Commands.CopyDeepLinkToBranch, ref);
+		}
+
+		return Promise.resolve();
```

Comment:
should we provide a notification for any of these copy actions if they do nothing?

Reply:
Maybe? I think we very often just log and fail today. It's definitely a balance.

Reply:
In this case I was following the setup we had set for other context menu options in the graph webview, but that's a good point. Maybe if we get user feedback about it we can change the standard and start giving notifications for failures with other commands too? Or should we consider this one an exception case where we definitely think the notification on failure is warranted?

Reply:
Makes sense to stay consistent. Sounds like there's a more holistic discussion around that if we want to change it.